aldryn_newsblog/managers.py

```diff
@@ -83,3 +82,10 @@ def get_tags(self, namespace):
         for tag in tags:
             tag.num_entries = counted_tags[tag.pk]
         return sorted(tags, key=attrgetter('num_entries'), reverse=True)
+
+
+class PublishedRelatedManager(RelatedManager):
+    def get_query_set(self):
+        qs = super(PublishedRelatedManager, self).get_query_set().filter(
+            is_published=True)
+        return qs
```

Comment:
Don't agree with add one more manager to Article model just to filter published items. We must have a published method in manager and queryset, something like that:

```python
from parler.managers import TranslatableManager, TranslatableQuerySet


class ArticleQuerySet(TranslatableQuerySet):

    def published(self):
        return self.filter(is_published=True)


class ArticleManager(TranslatableManager):

    def get_queryset(self):
        return ArticleQuerySet(self.model, using=self.db)

    def published(self):
        return self.get_query_set().published()
```

This way you can use `published` anytime in a query, for example:

```python
language = get_language_from_request(request, check_path=True)
events = Event.objects.language(language).translated(language)

if only_published:
    events = events.published()
```